### PR TITLE
Improve parallel-installation logging

### PR DESCRIPTION
**Pull request by colunira**
**Description**

Changes proposed in this pull request:

- Pass logging function as a configurable parameter
- Add log prefix to internal packages

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/9347

### CODE REVIEW

File: parallel-install/example/example.go [hunk 1]
```diff
@@ -2,8 +2,9 @@ package main
 
 import (
 	"flag"
+	log "github.com/sirupsen/logrus"
```
Comment (Tomasz-Smelcerz-SAP) [inline parallel-install/example/example.go:5]: Do we need logrus here? Perhaps built-in log package is enough?

File: parallel-install/pkg/installation/installation.go [hunk 2]
```diff
@@ -51,13 +50,17 @@ func NewInstallation(prerequisites [][]string, componentsYaml string, overridesY
 	}, nil
 }
 
+func (i *Installation) SetupLogger(log func(string, ...interface{})) {
```
Comment (Tomasz-Smelcerz-SAP) [inline parallel-install/pkg/installation/installation.go:53]: This function should be defined as top-level public function in the config package.

File: parallel-install/example/example.go (hunk 1 above)
Comment (colunira) [reply parallel-install/example/example.go:5]: I used it for testing purposes, I can remove it now :)

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Pull request merged**